docs/advanced-usage/env-support.md

```diff
@@ -52,7 +52,7 @@ DB_PORT=3306
 $config['save_tmpl_files'] = 'y';
 
 $config['base_url'] = $_ENV['BASE_URL'];
-$config['site_url'] = $config['base_url'];
+$config['site_url'] = $config['BASE_URL'];
```

Comment:
This isn't correct - on line 54 it's setting `$config['base_url']` (lowercase), so referencing it here with uppercase would be a different key.